index.html

```diff
@@ -1,276 +1,240 @@
+                </div>
+              </div>   
     </section>
-
     <section class="about">
-      <div class="container">
-        <div>
-          <h2 class="about_header">About<br />Myself</h2>
-          <p class="win">
-            Hello I’m a software developer! I can help you build a product ,
-            feature or website Look through some of my work and experience!
-            If you like what you see and have a project you need coded,
-            don’t hestiate to contact me.
-          </p>
-        </div>
-
-        <div>
-          <h2 class="media_header">LET'S CONNECT</h2>
-          <ul class="media_ul">
-            <li class="media_li"><a href="#" target="_blank"><img src="./images/Github-icon.svg" alt="Github icon"/></a>
-            </li>
-            <li class="media_li"><a href="#" target="_blank"><img src="./images/Linkedin-icon.svg" alt="Linkedin icon"/></a>
-            </li>
-            <li class="media_li"><a href="#" target="_blank"><img src="./images/Vector.svg" alt="Vector icon"/></a>
-            </li>
-            <li class="media_li"><a href="#" target="_blank"><img src="./images/Twitter-icon.svg" alt="Twitter icon"/></a>
-            </li>
-            <li class="media_li"><a href="#" target="_blank"><img src="./images/Medium-icon.svg" alt="Medium icon"/></a>
-            </li>
-          </ul>
-        </div>
-      </div> 
-      <a href="#" class="btn btn-mastery">Get my resume</a>
-      <div class="aim">
-        <div>
-          <ul>
-            <li aria-expanded="true" aria-haspopup="true">
-              Languages
-             
-              <i class="fa fa-angle-down"></i>
-            </li>
-          </ul>
-            <div class="about_img">
-              <img src="./images/webpng.png" alt="web image">
+        <div class="left-block">
+          <div>
+                <h2 class="about_header">About<br />Myself</h2>
+                <p class="about_text">
+                  Hello I’m a software developer! I can help you build a product ,
+                  feature or website Look through some of my work and experience!
+                  If you like what you see and have a project you need coded,
+                  don’t hestiate to contact me.
+                </p>
+            </div>
+              <div>
+                <h2 class="media_header">LET'S CONNECT</h2>
+                <ul class="media_ul">
+                  <li class="media_li"><a href="#" target="_blank"><img src="./images/Github-icon.svg" alt="Github icon"/></a>
+                  </li>
+                  <li class="media_li"><a href="#" target="_blank"><img src="./images/Linkedin-icon.svg" alt="Linkedin icon"/></a>
+                  </li>
+                  <li class="media_li"><a href="#" target="_blank"><img src="./images/Vector.svg" alt="Vector icon"/></a>
+                  </li>
+                  <li class="media_li"><a href="#" target="_blank"><img src="./images/Twitter-icon.svg" alt="Twitter icon"/></a>
```

Comment:
MOBILE VERSION:

- [ ] Kindly make sure the languages list is displayed as per the Figma design for the mobile version.
- [ ] Kindly make sure to reduce the spacing on the right side of the languages as per the Figma design as shown in the video.

Your design:


https://user-images.githubusercontent.com/70269507/166842668-dd4971d3-33c1-49ab-b003-bfe41f43da1f.mov



Figma design:
<img width="236" alt="Screenshot 2022-05-05 at 1 41 16 AM" src="https://user-images.githubusercontent.com/70269507/166842420-d694b476-fa54-4952-b811-398970ec8c2b.png">